modules/configuring-default-route-timeouts.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/config-applications/configuring-revision-timeouts.adoc
+:_content-type: PROCEDURE
+[id="configuring-default-route-timeouts_{context}"]
+= Configuring the default route timeouts per revision
+
+By configuring the route timeouts per revision, you can fine-tune timeouts for workloads with unique requirements, such as AI or data processing applications, without impacting the global timeout settings for other services.
+
+.Procedure
+
+You can configure the timeouts of a specific revision by updating your `KnativeServing` custom resource (CR), the Service definition, and using the `serving.knative.openshift.io/setRouteTimeout` annotation to adjust the OCP route timeout.
+
+. Set the `max-revision-timeout` as required:
```

Comment:
```suggestion
. Set the `max-revision-timeout` annotation as required:
```

Reply:
When you say `as required` do you mean that the user should know the value of this and set it according to their cluster's requirements. If so, I'd maybe reword to something like `Set the ... annotation to fit your requirements` or `as you require`.